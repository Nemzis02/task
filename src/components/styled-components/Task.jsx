import styled from 'styled-components';

export const Task = styled.div`
  display: flex;
  flex-direction: column;
  width: 800px;
  margin: auto;
`;
export const Content = styled.div`
  text-align: justify;
`;

export const Heading = styled.h1`
  color: #216298;
`;

export const BaseModalContainer = styled.div`
  width: 600px;
  border: 1px solid #b33a3a;
  background-color: #fff;
  border-radius: 10px;
`;
