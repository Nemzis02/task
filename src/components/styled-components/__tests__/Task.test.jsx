import React from 'react';
import { render } from '@testing-library/react';
import { Task } from '@components/styled-components/Task';
import 'jest-styled-components';

describe('Task styled component', () => {
  it('Should be rendered correctly', () => {
    const { container } = render(<Task />);
    expect(container).toMatchSnapshot();
  });
});
