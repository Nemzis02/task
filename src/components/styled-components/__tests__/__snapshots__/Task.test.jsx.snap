// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Task styled component Should be rendered correctly 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  width: 800px;
  margin: auto;
}

<div>
  <div
    class="c0"
  />
</div>
`;
